### DISCUSSION TITLE
Stand up Friday May 5th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#76] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Solved bug on login
- Worked on the Neo4J activity

**What are your priorities for the next 24 hours?**:
 - Still working on Neo4j
 - Finally make the upload of the freaking file to the back

**Blockers**:
 - None
 
**Shoutouts**:
 - @everyone for the presentation!

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Presentations

**What are your priorities for the next 24 hours?**:
 - Setup testing on back-end and frontend
 
**Blockers**:
 - None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 - Implemented suggested changes on my PR  
 
**What are your priorities for the next 24 hours?**:
- Deploy server to GC
- Develop CI/CD pipeline for deploying on every pull request

**Blockers**:
- None
 
**Shoutouts**:
 - @CristianCazares for keep working every day

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
 - Grind Leetcode

**What are your priorities for the next 24 hours?**:
 - Finish assignments for Sunday
 - Data analysis repo
 - Grind Leetcode

**Blockers**:
 - None
